ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -91,6 +94,11 @@ def main(argv=sys.argv[1:]):
         start_time = time.time()
 
     compilation_dbs = get_compilation_db_files(args.paths)
+    if args.packages_select:
+        if len(args.packages_select) == 1:
```

Comment:
Does this mean `nargs` should be `1` instead of `+`?

Reply:
What this does is handle the case where the user passes in a bash variable or a quoted list. In that case, the quoted list is put in the array as a single string with spaces separating it. This if statement breaks up that quoted list of package names into the list of package names we need for this change.